form/without_class.rst

```diff
@@ -112,3 +112,9 @@ but here's a short example::
     If the form is not mapped to an object, every object in your array of
     submitted data is validated using the ``Symfony\Component\Validator\Constraints\Valid``
     constraint, unless you :doc:`disable validation </form/disabling_validation>`.
+
+.. caution::
+
+    If a form is not submitted with the request (for example, when a form
+    is only partially submitted in a HTTP PATCH request), the constraints
+    configured for this particular form will be ignored.
```

Comment:
@xabbuh I think Javer's right, and the last sentence is confusing when you read: `, the constraints configured for this particular form will be ignored.` it's not clear enough what "constraints" you mean, the constraints defined in the root form or the constraints defined per form field.

Maybe we should revert the sentence and mention that "only the constraints from the submitted form fields will be evaluated" all others will be ignored.

Reply:
> When a form is only partially submitted (for example, in an HTTP PATCH request), only the constraints from the submitted form fields will be evaluated.

@yceruto @javiereguiluz Does that sound better?

Reply:
Yes! I now understand it perfectly. Thanks!